### DISCUSSION TITLE
How do I open the report with CLI if it's set to open "never" locally?

### DISCUSSION THREAD

**Discussion by alexzavg** [ortoniKC/ortoni-report#44] [General]
Hey,

Is there a way to open the report manually via CLI if the "open" param is set to "never"?

Something like with playwright's native HTML reporter which can be opened with the command `npx playwright open`? _(I've tried that btw and it opens the native HTML reporter 😅 )_

I haven't found that info in the repo description, so was just wondering if that's possible 👀

**Comment by ortoniKC** [comment]
https://youtu.be/kep8DCgDg6E?si=i1DdSr0I_9ooETBE
I have answered that in this video, pls check and suggest to me if you need any other solution.

**Comment by ortoniKC** [comment]
@alexzavg I have provided an option to open the report using npx ortoni-report show-report. 
Please check the latest version 2.0.9
